.github/workflows/wheels.yml

```diff
@@ -15,24 +15,24 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Install Python 3.11
+      - name: Install Python 3.12
         uses: actions/setup-python@v5
         with:
-          python-version: 3.11
+          python-version: 3.12
       - run: python -m pip install build
       - name: Build sdist
         run: python -m build --sdist
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
-          name: dist
+          name: dist-sdist
           path: dist/*.tar.gz
 
   build_wheels:
     name: Build Google Benchmark wheels on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, macos-13, macos-14, windows-latest]
```

Comment:
why both macoses?

Reply:
MacOS 13 builds Intel x86, MacOS 14 builds ARM64.

https://github.blog/changelog/2024-01-30-github-actions-macos-14-sonoma-is-now-available/

(i.e., both build their host architectures, which is x86_64 on macos-13 and arm64 on macos-14.)

Reply:
ah, nothing like obscuring architecture behind OS versions.